<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="runButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="WorkflowTreeWidget" name="workflowTree">
      <column>
       <property name="text">
        <string notr="true">Workflow</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="abortButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Abort</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="availablePlugins">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string notr="true">Available Extensions</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Workflow"/>
    <addaction name="actionSave_Workflow"/>
    <addaction name="separator"/>
    <addaction name="actionRun_Workflow"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Workflow">
   <property name="text">
    <string>Save Workflow</string>
   </property>
  </action>
  <action name="actionOpen_Workflow">
   <property name="text">
    <string>Open Workflow</string>
   </property>
  </action>
  <action name="actionRun_Workflow">
   <property name="text">
    <string>Run Workflow</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorkflowTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>workflowtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
